Source/Core/Core/PowerPC/JitArm64/JitArm64_RegCache.cpp

```diff
@@ -279,14 +281,25 @@ ARM64Reg Arm64FPRCache::R(u32 preg)
 	switch (reg.GetType())
 	{
 	case REG_REG: // already in a reg
+	case REG_LOWER_PAIR:
```

Comment:
This was intentional fall through to make the transition easier.
All cases are caught to be handled. It may be able to get changed in the future.